versions/3.0.4.md

```diff
@@ -89,11 +93,11 @@ The OpenAPI Specification is versioned using a `major`.`minor`.`patch` versionin
 
 Occasionally, non-backwards compatible changes may be made in `minor` versions of the OAS where impact is believed to be low relative to the benefit provided.
 
-An OpenAPI description document compatible with OAS 3.\*.\* contains a required [`openapi`](#oas-version) field which designates the version of the OAS that it uses.
+An OpenAPI Document compatible with OAS 3.\*.\* contains a required [`openapi`](#oas-version) field which designates the version of the OAS that it uses.
```

Comment:
Now that we have an "OpenAPI Document" section, I feel like this sentence belongs in that section.  When I read this from top to bottom just now the "OpenAPI Document" section felt vague, and this is the part that felt like it was missing.